spring-cloud-task-core/src/test/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDaoTests.java

```diff
@@ -166,6 +174,120 @@ public void testFindAllDefaultSort()  {
 		assertEquals("FOO3", taskExecution.getTaskName());
 	}
 
+	@Test
+	@DirtiesContext
+	public void testFindLatestTaskExecutions()  {
+		initializeRepositoryNotInOrderWithMultipleTaskExecutions();
+		final List<TaskExecution> latestTaskExecutions = dao.getLatestTaskExecutionsByTaskNames("FOO1");
+		assertTrue("Expected only 1 taskExecution but got " + latestTaskExecutions.size(), latestTaskExecutions.size() == 1);
+
+		final TaskExecution lastTaskExecution = latestTaskExecutions.get(0);
+		assertEquals("FOO1", lastTaskExecution.getTaskName());
+
+		final LocalDateTime dateTime = LocalDateTime.ofInstant(lastTaskExecution.getStartTime().toInstant(), ZoneOffset.UTC);
+
+		assertEquals(2015, dateTime.getYear());
+		assertEquals(Month.FEBRUARY, dateTime.getMonth());
+		assertEquals(22, dateTime.getDayOfMonth());
+		assertEquals(23, dateTime.getHour());
+		assertEquals(59, dateTime.getMinute());
+		assertEquals(0, dateTime.getSecond());
+	}
+
+	@Test
+	@DirtiesContext
+	public void testFindLatestTaskExecutions2()  {
```

Comment:
Can we change the name here to testFindLatestTaskExecutionsMultParams.  (or something like it)